### DISCUSSION TITLE
Difficulty adding to PATH

### DISCUSSION THREAD

**Discussion by tenebrosity** [Homebrew/discussions#1871] [Getting started]
I'm a newbie at coding, and I am trying to install homebrew, but it comes up with this: 

```
- Run these two commands in your terminal to add Homebrew to your PATH:
    echo 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> /Users/noellemallo/.zprofile
    eval "$(/opt/homebrew/bin/brew shellenv)"
```

So I enter those two commands separately in my terminal, and no text pops up. the `brew help` command comes up as 
`zsh: command not found: Example`

So I try entering the commands together, and the same thing happens, where no text pops up at all and it just gives me a new line.

I tried entering brew doctor and it says Your system is ready to brew.

What am I doing wrong?

**Comment by carlocab** [comment]
`brew` appears to be in your `PATH`, which is why `brew doctor` works just fine. The two commands you're meant to run to add `brew` to your `PATH` should not produce any output. (And you only need to run them once.)

What's the output of
```
brew config
which -a brew
wc -l ~/.zprofile
```
?
